### ISSUE TITLE
Swear filter Doesn't work with cbs.com

### ISSUE THREAD

**Issue by ShamanSquirrel** [richardfrost/AdvancedProfanityFilter#193]
I'm watching Star Trek Picard on CBS all access.  In my opinion, the F-word doesn't belong on Star Trek, period.  But the writers put it on the show just because they can get away with it since it's streaming.  Despite cbs.com being listed as one of the supported sites, it doesn't mute the audio when any cussing comes up.  And yes, I had the captions on with my replacement words.  It only replaced the captioned words, but it didn't mute the audio.  The service works on other sites such as Netflix.

**Comment by richardfrost** [comment]
Thanks for reporting it! I completely agree with you, it's sad they have crossed that line in Star Trek. Anyway, I will take a look tomorrow and see what's going wrong there. If you are available and willing, I may have you test something depending on what the problem and solution are.

**Comment by ShamanSquirrel** [comment]
I'm willing to test what needs to be tested, but I won't be around in the afternoon.

    On Monday, March 16, 2020, 10:36:45 PM CDT, Richard Frost <notifications@github.com> wrote:  
 
 


Thanks for reporting it! I completely agree with you, it's sad they have crossed that line in Star Trek. Anyway, I will take a look tomorrow and see what's going wrong there. If you are available and willing, I may have you test something depending on what the problem and solution are.

**Comment by richardfrost** [comment]
Alright, sorry it took so long. If you could, can you test this custom config and see if it works for you?

Put this in the custom audio sites section: Options page -> Audio tab

```json
{
  "www.cbs.com": [
    {
      "mode": "cue",
      "videoCueLanguage": "en",
      "videoCueRequireShowing": false
    }
  ]
}
```

**Comment by richardfrost** [comment]
@ShamanSquirrel Don't mean to rush you, but I'm just curious if you've had a chance to test this, or if you ran into any other trouble. Thanks!

**Comment by erichca** [comment]
I have the same issue and I tried the config and it works.

**Comment by richardfrost** [comment]
Perfect. I'll go ahead and put that change into the next version. Feel free to remove that custom code when you get the update if you want, but you can leave it in for now to enjoy the filter for now.

**Comment by ShamanSquirrel** [comment]
I put the code in, but I haven't tested it yet.  I'll let you know if it works!

    On Saturday, March 21, 2020, 5:24:27 PM CDT, Richard Frost <notifications@github.com> wrote:  
 
 


@ShamanSquirrel Don't mean to rush you, but I'm just curious if you've had a chance to test this, or if you ran into any other trouble. Thanks!

**Comment by ShamanSquirrel** [comment]
Okay, tested it. Seems to work now! Thanks! 
    On Saturday, March 21, 2020, 8:16:45 PM CDT, Richard Frost <notifications@github.com> wrote:  
 
 


Perfect. I'll go ahead and put that change into the next version. Feel free to remove that custom code when you get the update if you want, but you can leave it in for now to enjoy the filter for now.

**Comment by richardfrost** [comment]
Released with [2.6.1](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.6.1).

If you notice any trouble after getting the update, feel free to comment and re-open this issue.
